.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: yarn install
+      working-directory: ./backend
+      run: |
+        yarn install
+    - name: yarn test
+      working-directory: ./backend
+      run: |
+        yarn test 
+      env:
+        CI: true
+    - name: yarn build
+      working-directory: ./backend
+      run: |
+        yarn run build --if-present
+    - name: Add remote origin
+      working-directory: ./backend
+      run: |
+        git remote add heroku https://heroku:${{ secrets.HEROKU_API_KEY }}@git.heroku.com/${{ secrets.HEROKU_APP_NAME }}.git
```

Comment:
We'll need an Api key here, as well as the app name on Heroku in GH secrets under settings